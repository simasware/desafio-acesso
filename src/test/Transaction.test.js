test("Transaction with balance 1", async () => {
  expect(2).toBeGreaterThan(0);
});

test("Transaction with not enough balance 2", async () => {
  expect(2).toBeGreaterThan(0);
});

test("Transaction with negative value 3", async () => {
  expect(-2).toBeGreaterThan(0);
});

test("Transaction with negative value 4", async () => {
  expect("aaaa").toEqual(expect.any(Number));
});
