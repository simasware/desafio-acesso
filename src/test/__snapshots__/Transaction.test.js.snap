// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Transaction with balance 1`] = `
Object {
  "accountOrigin": "123",
  "accountDestination: "123",
  "value": 5.00
}
`;

exports[`Transaction with invalid value 4`] = `
Object {
  "accountOrigin": "123",
  "accountDestination: "123",
  "value": aaaaaa
}
`;

exports[`Transaction with negative value 3`] = `
Object {
  "accountOrigin": "123",
  "accountDestination: "123",
  "value": -2
}
`;

exports[`Transaction with not enough balance 2`] = `
Object {
  "accountOrigin": "123",
  "accountDestination: "123",
  "value": 65536
}
`;
